src/winconpty/winconpty.h

```diff
@@ -18,6 +18,7 @@ typedef struct _PseudoConsole
 // These are not defined publicly, but are used for controlling the conpty via
 //      the signal pipe.
 #define PTY_SIGNAL_CLEAR_WINDOW (2u)
+#define PTY_SIGNAL_REPARENT_WINDOW (3u)

```

Comment:
Remind me why it's 3?

Reply:
picked a random number this time. 2 & 8 had "reasons", this one - nah. Just random.

Reply:
Good enough.